data/convert.go

```diff
@@ -53,7 +53,10 @@ func NewWith(convert StructOptions, value interface{}) Value {
 	case reflect.String:
 		return String(v.String())
 	case reflect.Slice:
-		var slice []Value
+		if v.IsNil() {
+			return Null{}
```

Comment:
actually, seems like we should always return empty slice rather than null for reflect.Slice, no?  can just change the previous line to = []Value{} ?